commands/build.js

```diff
@@ -58,7 +58,8 @@ function buildLumin (argv, indexContent) {
     throw err;
   }
 
-  var buildCommand = `mabu -t ${argv.host ? 'host' : 'device'} ${packagePath}`;
+  var device = argv.debug !== false ? 'device' : 'release_device';
```

Comment:
Had to check for `debug` being `false` specifically, because if `debug` is undefined, it's assumed to be true.